### ISSUE TITLE
getter and setter in same object cause "no-dupe-keys"

### ISSUE THREAD

**Issue by psiphi75** [babel/babel-eslint#220]
I am running eslint 1.10.3 and babel-eslint 5.0.0-beta4.  When I run eslint on the code below (see config below that) it causes a `no-dupe-keys` error.  I don't expect any errors in this code, you should be able to have getters and setters.  I have tried this through the standard eslint parser and it does not have any issues.  I only get this error with babel-eslint.

test.js:
```javascript
'use strict';
var B = {
    get x () {
        return this.ecks;
    },
    set x (ecks) {
        this.ecks = ecks;
    }
};
B.x = 1;
```

eslint output:

    /home/me/test.js
      9:5  error  Duplicate key 'x'  no-dupe-keys
    
    ✖ 1 problem (1 error, 0 warnings)

Below is my `.eslintrc` configuration:

```javascript
    {
      "parser": "babel-eslint",
      "rules": {
        "no-dupe-keys": [2],
      }
    }
```

**Comment by hzoo** [comment]
Thanks - getters/setters aren't working. It's a known issue - similar to https://github.com/babel/babel-eslint/issues/218.

Also https://github.com/babel/babel-eslint/issues/199#issuecomment-161004937

**Comment by hzoo** [comment]
Published babel-eslint@5.0.0-beta6 which hopefully fixes this.
